index.js

```diff
@@ -79,7 +80,7 @@ const initialBooks = [
 

```

Comment:
- [ ] I think it would be better if we define the `initialBooks` array inside the class to avoid repetition and to make the code cleaner. see the structure below:
```js
class BookStore {
  constructor() {
    this.saveToLocalStorage = (data) => {
      const booksString = JSON.stringify(data);
      localStorage.setItem('bookStoreData', booksString);
      return true;
    };

    const rawBooksData = localStorage.getItem('bookStoreData');
    if (rawBooksData) {
      this.books = JSON.parse(rawBooksData);
    } else {
      this.books =  [ {
        title: 'the boy with wings',
        author: 'Basit Korede',
        id: generated(),
      },
      {
       title: 'Think Pythone',
       author: 'korede Basit',
       id: generated(),
      },
     ];
      this.saveToLocalStorage(this.books);
    }
  }
```

Reply:
The `initialBooks` array is not being repeated also see PR description.